src/app/mytools/my-tool/my-tool.component.html

```diff
@@ -143,7 +143,7 @@
                     class="accent-1-dark-btn"
                     type="button"
                     (click)="addToExistingTools()"
-                    [disabled]="(isRefreshing$ | async) || (userQuery.user$ | async) === false"
+                    [disabled]="(isRefreshing$ | async) || (userQuery.user$ | async) === null"
```

Comment:
The compiler started complaining about this, rightly so, `userQuery.user$`'s type is not a boolean. There are other similar instances of this that were also changed.